deltachat-ios/Controller/Settings/Proxy/ShareProxyViewController.swift

```diff
@@ -0,0 +1,142 @@
+import UIKit
+import DcCore
+import SDWebImageSVGKitPlugin
+
+class ShareProxyViewController: UIViewController {
```

Comment:
Maybe it would be worth to extract shared code from `ShareProxyViewController` and `QRCodeViewController` into a common component?

Reply:
there are some more changes done on android's contact-qr-controller anyways recently, let's see if and when we want that on iOS, then we can reconsider